client/packages/cli/index.js

```diff
@@ -554,18 +554,31 @@ async function createApp({ pkgDir, instantModuleName }) {
   }
 }
 
-async function promptForAppId() {
-  console.log(
-    "Great! Grab your app id from: " +
-      chalk.blueBright.underline("https://instantdb.com/dash"),
-  );
-  const _id = await input({
-    message: "What's the app ID?",
-    required: true,
-  }).catch(() => null);
-  const id = _id?.trim();
-  if (id) return id;
-  error("No app ID provided. Exiting.");
+async function promptForAppId(pkgAndAuthInfo) {
+  const res = await fetchJson({
+    debugName: "Fetching apps",
+    method: "GET",
+    path: "/dash",
+    errorMessage: "Failed to fetch apps.",
+  });
+  if (!res.ok) {
+    return;
+  }
+  const { apps } = res.data;
+  if (!apps.length) {
+    const ok = await "You don't have any apps. Want to create a new one?";
+    if (!ok) return;
+    await createApp(pkgAndAuthInfo);
+    return;
+  }
+  const choice = await select({
+    message: "Which app would you like to import?",
+    choices: res.data.apps.map((app) => {
+      return { name: app.title, value: app.id };
```

Comment:
Maybe we ought to do `${app.title} (${app.id})`?

Reply:
I like it!